format/spec.md

```diff
@@ -1436,6 +1457,7 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**                 | Not supported                                                                                                |
 | **`list`**                   | Not supported                                                                                                |
 | **`map`**                    | Not supported                                                                                                |
+| **`variant`**                | Not supported                                                                                                |
```

Comment:
I do agree this should be not-supported for now. Then when shredding is included say something like for Shredded variants only,  binary value concatenation of meatdata and value + separator byte or something. We can figure that out with the shredding addition though

Reply:
If we don't include Variant here, then we don't need to include it in the JSON section either.

Reply:
Seems binary representation is used for lower bound and upper bound and JSON single-value serialization is used for default value. Looks like they are defined independently? But since there is no default value for Variant, i will remove from JSON section.